.github/mvn-settings.xml

```diff
@@ -3,6 +3,22 @@
     <profile>
       <id>google-mirror</id>
       <repositories>
+        <repository>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <id>jboss-nexus</id>
+          <name>JBoss Nexus</name>
+          <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+        </repository>
+        <repository>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+          <id>jboss-staging-nexus</id>
+          <name>JBoss Staging Nexus</name>
+          <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
+        </repository>
```

Comment:
This will need to be removed before merging @rsynek

Reply:
Of course, I will remove it once our artifacts hit the Maven central.